lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -220,7 +220,7 @@ defmodule ExUnit.Runner do
     EM.test_started(config.manager, test)
 
     if is_nil(test.state) do
-      capture_log? = Map.get(tags, :capture_log, false)
+      capture_log? = Map.get(tags, :capture_log, Application.fetch_env!(:ex_unit, :capture_log))
```

Comment:
Actually, we should just store the default in the config. Take a look at get_timeout and how the timeout option is implemented. That's why I proposed to unify both of them under the tags option.